identity-wallet/src/state/core_utils/history_event.rs

```diff
@@ -14,7 +14,7 @@ pub enum EventType {
 #[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, TS)]
 #[ts(export)]
 pub struct HistoryEvent {
-    pub connection_id: Option<String>,
+    pub connection_id: String,
```

Comment:
Since this type has changed, we should also update the bindings :+1: 

Do you think there might also be some changes needed in the frontend once the binding is updated?

Reply:
No its not needed luckily